xgi/drawing/xgi_pylab.py

```diff
@@ -152,28 +193,32 @@ def draw_xgi_nodes(ax, H, pos, node_fc, node_ec, node_lw, node_size, zorder):
         )
 
 
-def draw_xgi_hyperedges(ax, H, pos, edge_lc, edge_lw, edge_fc, d_max):
+def draw_xgi_hyperedges(ax, H, pos, edge_lc, edge_lw, edge_fc, d_max, settings):
```

Comment:
Push "ax" to 3rd place to match draw()

Reply:
Describe possible keys for settings?
It's described in draw(), but people might want to use draw_xgi_hyperedges() and draw_xgi_nodes() independently.
Also, right now, I have to have all keys in settings, right? Want if I only want to change one, say the cmap.